scripts/build_demand_profiles.py

```diff
@@ -176,8 +191,42 @@ def build_demand_profiles(
     substation_lv_i = n.buses.index[n.buses["substation_lv"]]
     regions = gpd.read_file(regions).set_index("name").reindex(substation_lv_i)
     load_paths = load_paths
-    # Merge load .nc files: https://stackoverflow.com/questions/47226429/join-merge-multiple-netcdf-files-using-xarray
-    gegis_load = xr.open_mfdataset(load_paths, combine="nested")
+
+    if str(load_paths[0]).endswith(".csv"):
+        df = pd.concat(
+            [
+                pd.read_csv(
+                    pth,
+                    sep=";",
+                    dtype={
+                        "region_code": str,
+                        "time": str,
+                        "region_name": str,
+                        "Electricity demand": np.float64,
+                    },
+                    keep_default_na=False,
+                    na_values=["_", ""],
+                )
+                for pth in load_paths
+            ],
+            axis=0,
+        )
+        df.time = pd.to_datetime(df.time, format="%Y-%m-%d %H:%M:%S")
+        load_regions = {c: n for c, n in zip(df.region_code, df.region_name)}
+
+        gegis_load = df.set_index(["region_code", "time"]).to_xarray()
+        gegis_load = gegis_load.assign_coords(
+            {
+                "region_name": (
+                    "region_code",
+                    [name for (code, name) in load_regions.items()],
+                )
+            }
+        )
+    else:
+        # Merge load .nc files: https://stackoverflow.com/questions/47226429/join-merge-multiple-netcdf-files-using-xarray
+        gegis_load = xr.open_mfdataset(load_paths, combine="nested")
+
     gegis_load = gegis_load.to_dataframe().reset_index().set_index("time")
```

Comment:
The comments above assume that load_paths can contain any combination csv and/or nc files.
That requires to chenge the previous logic:
we may need to load each file independently either using a read_csv approach or a read_xarray, and then merge the xarrays with [xr.merge](https://docs.xarray.dev/en/stable/generated/xarray.merge.html)

Were you envisioning something like this?

The alternative only csv or only nc files is also possible, but requires a change in the logic on how the load_regions are loaded and may cause limitations unfortunately

Reply:
Great point. Have changed a logic to ensure that it's possible to use a combination of `csv` and `nc`-files

Reply:
For some reason here there is a ValueError, can you reproduce it locally?